swap_meet/vendor.py

```diff
@@ -1,2 +1,76 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        inventory = inventory if inventory is not None else []
+        self.inventory = inventory
```

Comment:
Very concise! You can also set self.inventory without first creating the `inventory` variable.

```python
self.inventory = inventory if inventory is not None else []
```